### DISCUSSION TITLE
ssh issue `terminals database is inaccessible`

### DISCUSSION THREAD

**Discussion by pumetu** [ghostty-org/ghostty#3638] [Q&A] [answered]
Hello,
I am having an issue where when I ssh into a machine and try to run the `clear` command I am getting `terminals database is inaccessible`. This seems to only happen on some machines that I ssh into.

**Comment by JakeWharton** [comment, marked as answer]
Did you follow https://ghostty.org/docs/help/terminfo yet?

**Comment by pumetu** [comment]
Yep following the help works, thanks!
